jupyter_server/services/api/api.yaml

```diff
@@ -441,14 +441,19 @@ paths:
       tags:
         - kernels
       parameters:
-        - name: name
+        - name: options
           in: body
-          description: Kernel spec name (defaults to default kernel spec for server)
           schema:
             type: object
+            required:
+              - name
             properties:
               name:
                 type: string
+                description: Kernel spec name (defaults to default kernel spec for server)
+              path:
+                type: string
+                description: path the cwd of the kernel
```

Comment:
- 'path **to** the cwd of the kernel'
- Looks like the other descriptions start with a capital letter, I recommend the same.
- This doesn't appear to be the place, but will the path always be Unix style like those expressed via the contents REST apis?

Reply:
Thanks for the insight and feedback, updated!